docs/application/tizen-studio/release-notes/2-4-release-notes.md

```diff
@@ -2,39 +2,38 @@
 
 -   Release Date: May 29, 2018
 
-
 ## IDE and Tools
 
 
 ### New Features
 
 -   Device Manager
-	-   Enabled Device Manager Context UI to be extended by plugins
--   RT-IDE
-	-   RT-IDE GNU Compiler Collection (GCC) toolchain is updated to version 6.3.1.
+	-   Enabled Device Manager Context UI has been extended by plugins.
+-   RT IDE
+	-   RT IDE GNU Compiler Collection (GCC) toolchain has been updated to version 6.3.1.
 -   SmartThings&trade;
-	-   The setupId is a mandatory field and is mapped to the device onboarding ID.
-	-   The RESTful API Modeling Language (RAML) for SmartThings&trade; is updated.
+	-   The setupId is a mandatory field and has been mapped to the device onboarding ID.
+	-   The RESTful API Modeling Language (RAML) for SmartThings&trade; has been updated.
 -   Package Manager
-	-   Removed portable object (*.po) files , which are packaged into tpk.
+	-   The portable object (*.po) files has been removed, which are packaged into tpk.
```

Comment:
Just a request to check the tense of this sentence.
"has bee pp" + "present". Is it ok?

Reply:
@mijin85cho, Thanks for pointing it out. It is incorrect to use "has been pp" + "present"
 I will be changing the sentence as:
The portable object (*.po) files, which are packaged into tpk have been removed.